Px.Utils/ModelBuilders/MatrixMetadataBuilder.cs

```diff
@@ -31,21 +31,61 @@ public MatrixMetadataBuilder(PxFileSyntaxConf? pxFileSyntaxConf = null)
         }
 
         /// <summary>
-        /// Builds a MatrixMetadata object from a given set of metadata entries.
+        /// Builds a <see cref="MatrixMetadata"/> object from a given set of metadata entries.
```

Comment:
Riittäisikö dokumentointi vain interfacessa, nyt on duplikointia ja mahdollisesti eri dokumentointi interfacessa ja totetuttavassa luokassa.  tai ehkä vain /// <inheritdoc />

Sama kommentti myös muihin toteutuksiin.

Reply:
The original intention was to provide more implementation spesific information in the implementation documentation. But you are right that in these cases it's mostly just duplication. Using inheritdoc in the future when there is not anything special worth documenting in the implementation would be a good idea.